src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -19,9 +20,17 @@ export class InstructorHelpPageComponent implements OnInit {
 
   @ViewChild('helpPage') bodyRef ?: ElementRef;
 
-  constructor() { }
+  constructor(private route: ActivatedRoute) { }
 
   ngOnInit(): void {
+    this.route.fragment.subscribe((f: string) => {
+      setTimeout((): void => {
```

Comment:
Is it possible to use the `scroll` method https://github.com/TEAMMATES/teammates/pull/9696/files#diff-3387964a13f3e1193898cc714d80d931R50 ?

Reply:
Yes, I've updated that.

Reply:
Do you really need the `setTimeout()`? If it is just to wait for Angular to render the page, there is a better way to do it.

Reply:
I've changed to use `ngAfterViewInit()` lifecycle hook, so no need the `setTimout()`